1.18.2デバッグ: LuckPermsの権限読み込みエラー

```
[22:29:18 INFO]: [LuckPerms] Performing initial data load...
[22:29:18 WARN]: me.lucko.luckperms.common.storage.implementation.file.FileIOException: Exception thrown whilst reading/writing file: admin
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.implementation.file.AbstractConfigurateStorage.loadGroup(AbstractConfigurateStorage.java:278)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.util.Iterators.tryIterate(Iterators.java:41)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.implementation.file.SeparatedConfigurateStorage.loadAllGroups(SeparatedConfigurateStorage.java:304)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.implementation.split.SplitStorage.loadAllGroups(SplitStorage.java:175)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.Storage.lambda$loadAllGroups$11(Storage.java:202)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.Storage.lambda$future$1(Storage.java:98)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.ForkJoinTask.doExec(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.ForkJoinPool.scan(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.ForkJoinPool.runWorker(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source)
[22:29:18 WARN]: Caused by: java.io.IOException: while scanning an alias
[22:29:18 WARN]:  in 'reader', line 4, column 3:
[22:29:18 WARN]:     - *
[22:29:18 WARN]:       ^
[22:29:18 WARN]: unexpected character found
[22:29:18 WARN]: (10)
[22:29:18 WARN]:  in 'reader', line 4, column 4:
[22:29:18 WARN]:     - *
[22:29:18 WARN]:        ^
[22:29:18 WARN]:
[22:29:18 WARN]: 	at me.lucko.luckperms.lib.configurate.loader.AbstractConfigurationLoader.load(AbstractConfigurationLoader.java:146)
[22:29:18 WARN]: 	at me.lucko.luckperms.lib.configurate.loader.ConfigurationLoader.load(ConfigurationLoader.java:61)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.implementation.file.SeparatedConfigurateStorage.readFile(SeparatedConfigurateStorage.java:116)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.implementation.file.SeparatedConfigurateStorage.readFile(SeparatedConfigurateStorage.java:105)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.implementation.file.AbstractConfigurateStorage.loadGroup(AbstractConfigurateStorage.java:269)
[22:29:18 WARN]: 	... 12 more
[22:29:18 WARN]: Caused by: while scanning an alias
[22:29:18 WARN]:  in 'reader', line 4, column 3:
[22:29:18 WARN]:     - *
[22:29:18 WARN]:       ^
[22:29:18 WARN]: unexpected character found
[22:29:18 WARN]: (10)
[22:29:18 WARN]:  in 'reader', line 4, column 4:
[22:29:18 WARN]:     - *
[22:29:18 WARN]:        ^
[22:29:18 WARN]:
[22:29:18 WARN]: 	at org.yaml.snakeyaml.scanner.ScannerImpl.scanAnchor(ScannerImpl.java:1515)
[22:29:18 WARN]: 	at org.yaml.snakeyaml.scanner.ScannerImpl.fetchAlias(ScannerImpl.java:957)
[22:29:18 WARN]: 	at org.yaml.snakeyaml.scanner.ScannerImpl.fetchMoreTokens(ScannerImpl.java:406)
[22:29:18 WARN]: 	at org.yaml.snakeyaml.scanner.ScannerImpl.checkToken(ScannerImpl.java:251)
[22:29:18 WARN]: 	at org.yaml.snakeyaml.parser.ParserImpl$ParseIndentlessSequenceEntryValue.produce(ParserImpl.java:603)
[22:29:18 WARN]: 	at org.yaml.snakeyaml.parser.ParserImpl$ParseIndentlessSequenceEntryKey.produce(ParserImpl.java:586)
[22:29:18 WARN]: 	at org.yaml.snakeyaml.parser.ParserImpl.peekEvent(ParserImpl.java:166)
[22:29:18 WARN]: 	at org.yaml.snakeyaml.parser.ParserImpl.checkEvent(ParserImpl.java:156)
[22:29:18 WARN]: 	at org.yaml.snakeyaml.composer.Composer.composeSequenceNode(Composer.java:240)
[22:29:18 WARN]: 	at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:186)
[22:29:18 WARN]: 	at org.yaml.snakeyaml.composer.Composer.composeValueNode(Composer.java:314)
[22:29:18 WARN]: 	at org.yaml.snakeyaml.composer.Composer.composeMappingChildren(Composer.java:305)
[22:29:18 WARN]: 	at org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:286)
[22:29:18 WARN]: 	at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:188)
[22:29:18 WARN]: 	at org.yaml.snakeyaml.composer.Composer.getNode(Composer.java:115)
[22:29:18 WARN]: 	at org.yaml.snakeyaml.composer.Composer.getSingleNode(Composer.java:142)
[22:29:18 WARN]: 	at org.yaml.snakeyaml.constructor.BaseConstructor.getSingleData(BaseConstructor.java:151)
[22:29:18 WARN]: 	at org.yaml.snakeyaml.Yaml.loadFromReader(Yaml.java:491)
[22:29:18 WARN]: 	at org.yaml.snakeyaml.Yaml.load(Yaml.java:442)
[22:29:18 WARN]: 	at me.lucko.luckperms.lib.configurate.yaml.YAMLConfigurationLoader.loadInternal(YAMLConfigurationLoader.java:133)
[22:29:18 WARN]: 	at me.lucko.luckperms.lib.configurate.loader.AbstractConfigurationLoader.load(AbstractConfigurationLoader.java:137)
[22:29:18 WARN]: 	... 16 more
[22:29:18 WARN]: me.lucko.luckperms.common.storage.implementation.file.FileIOException: Exception thrown whilst reading/writing file: observer
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.implementation.file.AbstractConfigurateStorage.loadGroup(AbstractConfigurateStorage.java:278)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.util.Iterators.tryIterate(Iterators.java:41)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.implementation.file.SeparatedConfigurateStorage.loadAllGroups(SeparatedConfigurateStorage.java:304)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.implementation.split.SplitStorage.loadAllGroups(SplitStorage.java:175)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.Storage.lambda$loadAllGroups$11(Storage.java:202)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.Storage.lambda$future$1(Storage.java:98)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.ForkJoinTask.doExec(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.ForkJoinPool.scan(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.ForkJoinPool.runWorker(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source)
[22:29:18 WARN]: Caused by: java.io.IOException: Is a directory
[22:29:18 WARN]: 	at java.base/sun.nio.ch.FileDispatcherImpl.read0(Native Method)
[22:29:18 WARN]: 	at java.base/sun.nio.ch.FileDispatcherImpl.read(Unknown Source)
[22:29:18 WARN]: 	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
[22:29:18 WARN]: 	at java.base/sun.nio.ch.IOUtil.read(Unknown Source)
[22:29:18 WARN]: 	at java.base/sun.nio.ch.IOUtil.read(Unknown Source)
[22:29:18 WARN]: 	at java.base/sun.nio.ch.FileChannelImpl.read(Unknown Source)
[22:29:18 WARN]: 	at java.base/sun.nio.ch.ChannelInputStream.read(Unknown Source)
[22:29:18 WARN]: 	at java.base/sun.nio.ch.ChannelInputStream.read(Unknown Source)
[22:29:18 WARN]: 	at java.base/sun.nio.ch.ChannelInputStream.read(Unknown Source)
[22:29:18 WARN]: 	at java.base/sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
[22:29:18 WARN]: 	at java.base/sun.nio.cs.StreamDecoder.implRead(Unknown Source)
[22:29:18 WARN]: 	at java.base/sun.nio.cs.StreamDecoder.read(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.io.InputStreamReader.read(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.io.BufferedReader.fill(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.io.BufferedReader.read1(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.io.BufferedReader.read(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.io.Reader.read(Unknown Source)
[22:29:18 WARN]: 	at me.lucko.luckperms.lib.configurate.loader.CommentHandlers.beginsWithPrefix(CommentHandlers.java:244)
[22:29:18 WARN]: 	at me.lucko.luckperms.lib.configurate.loader.CommentHandlers$AbstractPrefixHandler.extractHeader(CommentHandlers.java:196)
[22:29:18 WARN]: 	at me.lucko.luckperms.lib.configurate.loader.CommentHandlers.extractHeader(CommentHandlers.java:70)
[22:29:18 WARN]: 	at me.lucko.luckperms.lib.configurate.loader.CommentHandlers.extractComment(CommentHandlers.java:91)
[22:29:18 WARN]: 	at me.lucko.luckperms.lib.configurate.loader.AbstractConfigurationLoader.load(AbstractConfigurationLoader.java:131)
[22:29:18 WARN]: 	at me.lucko.luckperms.lib.configurate.loader.ConfigurationLoader.load(ConfigurationLoader.java:61)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.implementation.file.SeparatedConfigurateStorage.readFile(SeparatedConfigurateStorage.java:116)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.implementation.file.SeparatedConfigurateStorage.readFile(SeparatedConfigurateStorage.java:105)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.implementation.file.AbstractConfigurateStorage.loadGroup(AbstractConfigurateStorage.java:269)
[22:29:18 WARN]: 	... 12 more
[22:29:18 WARN]: java.util.concurrent.CompletionException: java.lang.RuntimeException: Exception occurred whilst loading a group
[22:29:18 WARN]: 	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.ForkJoinTask.doExec(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.ForkJoinPool.scan(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.ForkJoinPool.runWorker(Unknown Source)
[22:29:18 WARN]: 	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source)
[22:29:18 WARN]: Caused by: java.lang.RuntimeException: Exception occurred whilst loading a group
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.implementation.file.SeparatedConfigurateStorage.loadAllGroups(SeparatedConfigurateStorage.java:305)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.implementation.split.SplitStorage.loadAllGroups(SplitStorage.java:175)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.Storage.lambda$loadAllGroups$11(Storage.java:202)
[22:29:18 WARN]: 	at me.lucko.luckperms.common.storage.Storage.lambda$future$1(Storage.java:98)
[22:29:18 WARN]: 	... 7 more
```

--- a/seichi-onp-k8s/manifests/seichi-kubernetes/app-templates/seichi-debug-minecraft-on-seichiassist-pr/templates/common-configs/luckperms-permission-configs.yaml
+++ b/seichi-onp-k8s/manifests/seichi-kubernetes/app-templates/seichi-debug-minecraft-on-seichiassist-pr/templates/common-configs/luckperms-permission-configs.yaml
@@ -303,7 +303,8 @@ data:
     name: admin
     permissions:
     # TODO: ホワイトリスト方式にする（ワイルドカードで権限を与えない）
-    - *
+    # NOTE: クォーテーションで囲まないと読み込んでくれない
+    - '*'
     - minecraft.command.deop:
         value: false
     - minecraft.command.op:

